packages/react/src/components/builder-page.component.tsx

```diff
@@ -853,8 +853,8 @@ export class BuilderPage extends React.Component<
                         ...this.props.options
                       }}
                       inline={
-                        !Builder.isEditing &&
-                        (this.props.inlineContent || 'content' in this.props)
+                        this.props.inlineContent ||
+                        (!Builder.isEditing && 'content' in this.props)
```

Comment:
great find, how did you know to look here?

Reply:
I did a diff of the last web components version that worked with the current one and went through the code to see what could have done it. Plus I was familiar because I fixed a different bug that had to do with `inline`, and I introduced this particular bug with my last fix 😄